index.bs

```diff
@@ -5830,6 +5830,74 @@ At this time, one [=credential property=] is defined: the [=resident key credent
+
+Since a certificate system needs to sign over the public key of the credential, and that public key is only available after creation, this extension is not defined in the [=registration extension|registration=] context.
+
+Since certificates are sizable relative to the storage capabilities of typical authenticators, user agents SHOULD consider what indications and confirmations are suitable to best guide the user in allocating this limited resource and prevent abuse.
+
+: Extension identifier
+:: `largeBlob`
+
+: Operation applicability
+:: [=authentication extension|Authentication=]
+
+: Client extension input
+::  <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        ArrayBuffer largeBlobWrite;
+        boolean largeBlobRead;
+    };
+    </xmp>
+
+    <div dfn-type="dict-member" dfn-for="AuthenticationExtensionsClientInputs">
+        :   <dfn>largeBlobWrite</dfn>
+        ::  An opaque byte string that the [=[RP]=] wishes to store with the existing credential.
+
+        :   <dfn>largeBlobRead</dfn>
+        ::  A boolean that indicates that the [=[RP]=] would like to fetch the previously-written blob associated with the asserted credential.
+    </div>
+
+: Client extension processing ([=authentication extension|authentication=])
+::
+       1. If {{AuthenticationExtensionsClientInputs/largeBlobRead}} is present and has the value `true`:
+           1. If the [[#sctn-getAssertion|assertion]] operation is successful, attempt to read any data associated with the asserted credential.
+           1. If successful, set {{AuthenticationExtensionsClientOutputs/largeBlob}} to the result.
+       1. If {{AuthenticationExtensionsClientInputs/largeBlobWrite}} is present, {{PublicKeyCredentialRequestOptions/allowCredentials}} contains exactly one element, and {{AuthenticationExtensionsClientInputs/largeBlobRead}} is not present, or has the value `false`:
+           1. If the [[#sctn-getAssertion|assertion]] operation is successful, attempt to store the contents of {{AuthenticationExtensionsClientInputs/largeBlobWrite}} on the [=authenticator=], associated with the indicated credential.
+           1. Set {{AuthenticationExtensionsClientOutputs/largeBlobWritten}} to `true` if successful and `false` otherwise.
+
+: Client extension output
+:: <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientOutputs {
+        ArrayBuffer largeBlob;
+        boolean largeBlobWritten;
+    };
+    </xmp>
+
+    <div dfn-type="dict-member" dfn-for="AuthenticationExtensionsClientOutputs">
+        :   <dfn>largeBlob</dfn>
+        ::  The opaque byte string that was associated with the credential identified by {{PublicKeyCredential/rawId}}. Only valid in the [=authentication extension=] case and only present if {{AuthenticationExtensionsClientInputs/largeBlobRead}} was `true`.
```

Comment:
>Only valid in the authentication extension case

This seems redundant since the extension is already defined as an authentication extension only.

Reply:
Agreed. Removed.